### ISSUE TITLE
[데이터셋검토] URBAN-SED

### ISSUE THREAD

**Issue by leemgs** [aigovsensing/dataset-review#188]
### 데이터셋 명칭

URBAN-SED

### 데이터셋 저장소 URL

http://urbansed.weebly.com/

### 추가 참고 사항

URBAN-SED is a dataset of 10,000 soundscapes with sound event annotations generated using the scraper library. The dataset includes 10,000 soundscapes, totals almost 30 hours and includes close to 50,000 annotated sound events. Every soundscape is 10 seconds long and has a background of Brownian noise resembling the typical hum often heard in urban environments. Every soundscape contains between 1-9 sound evnts from the following classes: air_conditioner, car_horn, children_playing, dog_bark, drilling, engine_idling, gun_shot, jackhammer, siren and street_music. The source material for the sound events are the clips from the UrbanSound8K dataset. URBAN-SED comes pre-sorted into three sets: train, validate and test. There are 6000 soundscapes in the training set, generated using clips from folds 1-6 in UrbanSound8K, 2000 soundscapes in the validation set, generated using clips from fold 7-8 in UrbanSound8K, and 2000 soundscapes in the test set, generated using clips from folds 9-10 in UrbanSound8K. Source: [http://urbansed.weebly.com/](http://urbansed.weebly.com/) Image Source: [http://urbansed.weebly.com/](http://urbansed.weebly.com/)
